.github/scripts/setup.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+go get github.com/alecthomas/gometalinter && gometalinter --install
+go generate . ./operations
+go install . ./operations
```

Comment:
there's a reason that non-binary, non-empty files should always have newline at the end of file. could you fix this? https://thoughtbot.com/blog/no-newline-at-end-of-file